sycl/doc/extensions/proposed/sycl_ext_oneapi_global_defaults.asciidoc

```diff
@@ -0,0 +1,150 @@
+= sycl_ext_oneapi_global_defaults
```

Comment:
This name doesn't seem right anymore.  Maybe rename it to "sycl_ext_oneapi_current_device"?  This will affect:

* The name here
* The name of the feature-test macro
* The name of the ".asciidoc" file